src/components/header/Header.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+
+function RenderHeader() {
+  return (
+    <Header>
+      <h1>Allaround Family</h1>
+    </Header>
+  )
+}
+
+const Header = styled.div`
+  width: 100%;
+  padding: 20px 0px;
+  text-align: center;
+  z-index: 2;
+  position: fixed;
+  top: 0;
```

Comment:
You'll probably need some kind of background when scrolling. 
Tip: experiment with `backdrop-filter` e.g. `backdrop-filter: blur(10px);`
Saw that here: https://blog.workflowy.com/ 
;)

Reply:
For Firefox, you'll need a workaround though, see: caniuse.com and  https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter

Reply:
This element has a slightly opaque background-color now.